### ISSUE TITLE
[JENKINS-71497] WCAG Operable:2.1 - Keyboard operation and navigation - Result 5. Some content is unusually operable for keyboard users.

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#6157]
<p>On the overview page for the branches within a repository, various contents are displayed in a table below a tab register structure. Navigation is done exclusively with the tab key, which means that keyboard users have to navigate through all tab registers to get to the content in the tab panels. This way of navigating is not efficient.</p>

<p><b>Suggested fix:</b></p>
<ul>
	<li>The tab registers are usually operated with the keyboard in such a way that the first register is jumped to with the tabulator key, navigation between the tab registers is carried out with the arrow keys, while changing to the associated tab panel with the tabulator key button is done. This allows efficient navigation between the tab registers and the corresponding tab panels and the tab order is consistent.</li>
	<li>Note: Screen reader users cannot perceive the roles of the tab registers correctly, as the implementation uses input fields of type "Radio" with integrated links. See also result 9. → For a barrier-free implementation of tab registers, both the roles, attributes and in particular the keyboard operation must be converted to tabs in accordance with the WAI ARIA Authoring Practices.3</li>
</ul>

---
<details><summary><i>Originally reported by <a title="Cristina Pizzagalli" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cristinap">cristinap</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-71497" target="_blank">WCAG Operable:2.1 - Keyboard operation and navigation - Result 5. Some content is unusually operable for keyboard users.</a></i></summary>
<i><ul>
<li><b>reported by</b>: cristinap
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>
